Added Event entity in weatherflow.markdown (#39557)
home-assistant/home-assistant.github.io: source/_integrations/weatherflow.markdown

- Wind lull
- Wind speed

## Event entity

> The Weatherflow tempest also sends out event triggers when it starts raining and when there is a lightning strike near by.

Reviewer: ```suggestion
The WeatherFlow tempest also sends out event triggers when it starts raining and when there is a lightning strike near by.
```
What is WeatherFlow tempest? Do you mean the WeatherFlow integration?

Author: a tempest is a device of weatherflow, not all devices have these events published

> The WeatherFlow tempest also sends out event triggers when it starts raining and when there is a lightning strike near by.

Reviewer: ```suggestion
The WeatherFlow Tempest station also sends out event triggers when it starts raining and when there is a lightning strike near by.
```
should Tempest be capital then? Is it a brand name? and add station or device to make it clear what it is?

Author: Agreed, its a type name, weatherflow is the brand.

> This {% term integration %} will expose these {% term event entity %} and can be used for automations the following entity's will be exposed:

Reviewer: ```suggestion
This {% term integration %} will expose these {% term event %} {% term entities %}  and can be used for automations. The following entities will be exposed:
```
